Configurations/10-main.conf

```diff
@@ -1624,13 +1660,12 @@ my %targets = (
     "darwin-common" => {
         inherit_from     => [ "BASE_unix" ],
         template         => 1,
-        cc               => "cc",
-        cflags           => picker(default => "",
-                                   debug   => "-g -O0",
+        CC               => "cc",
+        CFLAGS           => picker(debug   => "-g -O0",
                                    release => "-O3"),
         cppflags         => threads("-D_REENTRANT"),
+        ldflags          => "-Wl,-search_paths_first",
```

Comment:
I suppose this is typo, as we already established this flag should sting the eye as `lflags` :-) ... Yes, attempt to run ./Configure darwin-ppc-cc yields Makfile without -Wl,-search_paths_first